### PR TITLE
sfml: add version 2.6.2

### PR DESCRIPTION
**Pull request by toge**
### Summary
Changes to recipe:  **sfml/2.6.2**

#### Motivation
There are lots of bugfixesin 2.6.2.

#### Details
https://github.com/SFML/SFML/compare/2.6.1...2.6.2

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

**Review by ErniGH — APPROVED** [review]
LGTM

**Review by perseoGI — DISMISSED** [review]
LGTM https://github.com/SFML/SFML/compare/2.6.1...2.6.2
I see you have merged in the `2.6.2-0001-cmake-robust-find-deps.patch` the diff off `2.6.0-0007-use-cci-minimp3.patch` and upstream has included natively the `2.6.0-0004-fix-ios.patch`

So perfect! 🐸

File: recipes/sfml/all/conanfile.py [hunk 1]
```diff
@@ -99,6 +99,12 @@ def generate(self):
             tc.variables["SFML_USE_STATIC_STD_LIBS"] = is_msvc_static_runtime(self)
         tc.generate()
         deps = CMakeDeps(self)
+        if Version(self.version) >= "2.6.2":
```
Comment (AbrilRBS) [inline recipes/sfml/all/conanfile.py:102]: @toge this approach for Conan 2 is better than patch the cmakelists :)

**Review by AbrilRBS — APPROVED** [review]
Thanks! sorry for the noise

**Review by perseoGI — APPROVED** [review]
Eyes pleasure!

**Review by ErniGH — APPROVED** [review]

**Pull request merged**